templates/Pages/about.php

```diff
@@ -41,6 +41,25 @@
         <li><strong>IO500 List</strong>: the subset from the full list with only the best submission per storage system</li>
         <li><strong>10 Node Challenge List</strong>: the subset from the full list with only the best submission per storage system ran on exactly ten nodes</li>
     </ul>
+    
+    <p>
+        Please note that the Ranked Lists will effectively <i>hide</i> some submissions due to the fact that they only show the best submission per storage systems.  Therefore, 
+        if a storage system has multiple submissions, only the top one by overall score will be visible in the Ranked Lists but all submissions will appear in the other lists.  
+        This can sometimes result in seemingly confusing lists so it is important to understand this distinction.  For example, consider the following three submissions:
+        <a href="/site/submissions/view/27">Submission 27 from the JCAHPC Oakforest-PACS IME system</a>,      <!-- score 168.09 bw	581.43 -->
```

Comment:
Are the "score" and "bw" numbers shown in the text?  They look like comments that will only be visible in the HTML but not displayed.

I think it would be more useful if the actual values were shown on the page, and also include the "client_nodes" data to make it clear why there are two entries and why the performance is different.

Reply:
I think a good way to show this would be to have a little (static) table showing these results:
![example-full](https://user-images.githubusercontent.com/167584/99846877-605e0400-2b34-11eb-8c82-8796a2764758.png)